boards/bouffalolab/bl60x/bl604e_iot_dvk/doc/index.rst

```diff
@@ -0,0 +1,167 @@
+.. zephyr:board:: bl604e_iot_dvk
+
+Overview
+********
+
+BL602/BL604 is a Wi-Fi+BLE chipset introduced by Bouffalo Lab, which is used
+for low power consumption and high performance application development.  The
+wireless subsystem includes 2.4G radio, Wi-Fi 802.11b/g/n and BLE 5.0
+baseband/MAC design.  The microcontroller subsystem includes a 32-bit RISC CPU
+with low power consumption, cache and memory.  The power management unit
+controls the low power consumption mode.  In addition, it also supports
+various security features.  The external interfaces include SDIO, SPI, UART,
+I2C, IR remote, PWM, ADC, DAC, PIR and GPIO.
+
+The BL602 Development Board features a SiFive E24 32 bit RISC-V CPU with FPU,
+it supports High Frequency clock up to 192Mhz, have 128k ROM, 276kB RAM,
+2.4 GHz WIFI 1T1R mode, support 20 MHz, data rate up to 72.2 Mbps, BLE 5.0
+with 2MB phy.  It is a secure MCU which supports Secure boot, ECC-256 signed
+image, QSPI/SPI Flash On-The-Fly AES Decryption and PKA (Public Key
+Accelerator).
+
+Hardware
+********
+
+For more information about the Bouffalo Lab BL-60x MCU:
+
+- `Bouffalo Lab BL60x MCU Website`_
+- `Bouffalo Lab BL60x MCU Datasheet`_
+- `Bouffalo Lab Development Zone`_
+- `The RISC-V BL602 Book`_
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+The default configuration can be found in the Kconfig
+:zephyr_file:`boards/bouffalolab/bl60x/bl604e_iot_dvk/bl604e_iot_dvk_defconfig`.
+
+System Clock
+============
+
+The BL604E Development Board is configured to run at max speed (192MHz).
+
+Serial Port
+===========
+
+The ``bl604e_iot_dvk`` board uses UART0 as default serial port.  It is connected
+to USB Serial converter and port is used for both program and console.
+
+
+Programming and Debugging
+*************************
+
+BL Flash tool
```

Comment:
This won't scale well. I assume that when you add support for other boards in future PRs, you will extract this common information to somewhere in doc/develop/flash_debug/host-tools.rst, and link to it from here?

If so, no problem to get it done quickly like this for the initial PR, though it'd be nice to get it done that way from the start.

Reply:
Indeed,

Just to mention, this tool will be replaced. We are looking alternatives and working to fully support OpenOCD.
This is the motive I prefer keep this way at moment.

Reply:
removed from PR